diff --git a/node_modules/react-native-datepicker/style.js b/node_modules/react-native-datepicker/style.js
index 30d2cc6..e6359b9 100644
--- a/node_modules/react-native-datepicker/style.js
+++ b/node_modules/react-native-datepicker/style.js
@@ -1,45 +1,46 @@
-import {StyleSheet} from 'react-native';
+import {StyleSheet} from 'react-native'
 
 let style = StyleSheet.create({
   dateTouch: {
-    width: 142
+    width: 142,
   },
   dateTouchBody: {
     flexDirection: 'row',
     height: 40,
     alignItems: 'center',
-    justifyContent: 'center'
+    justifyContent: 'center',
   },
   dateIcon: {
     width: 32,
     height: 32,
     marginLeft: 5,
-    marginRight: 5
+    marginRight: 5,
   },
   dateInput: {
     flex: 1,
     height: 40,
-    borderWidth: 1,
-    borderColor: '#aaa',
+
     alignItems: 'center',
-    justifyContent: 'center'
+    justifyContent: 'center',
   },
   dateText: {
-    color: '#333'
+    color: '#000',
+    fontWeight: 'bold',
+    marginStart: 20,
   },
   placeholderText: {
-    color: '#c9c9c9'
+    color: '#c9c9c9',
   },
   datePickerMask: {
     flex: 1,
     alignItems: 'flex-end',
     flexDirection: 'row',
-    backgroundColor: '#00000077'
+    backgroundColor: '#00000077',
   },
   datePickerCon: {
     backgroundColor: '#fff',
     height: 0,
-    overflow: 'hidden'
+    overflow: 'hidden',
   },
   btnText: {
     position: 'absolute',
@@ -48,29 +49,26 @@ let style = StyleSheet.create({
     paddingHorizontal: 20,
     flexDirection: 'row',
     alignItems: 'center',
-    justifyContent: 'center'
+    justifyContent: 'center',
   },
   btnTextText: {
     fontSize: 16,
-    color: '#46cf98'
+    color: '#000',
+    fontWeight: 'bold',
   },
   btnTextCancel: {
-    color: '#666'
+    color: '#666',
   },
   btnCancel: {
-    left: 0
+    left: 0,
   },
   btnConfirm: {
-    right: 0
+    right: 0,
   },
   datePicker: {
     marginTop: 42,
-    borderTopColor: '#ccc',
-    borderTopWidth: 1
   },
-  disabled: {
-    backgroundColor: '#eee'
-  }
-});
+  disabled: {},
+})
 
-export default style;
+export default style
